docs/ml-features.md

```diff
@@ -654,7 +654,11 @@ for expanded in polyDF.select("polyFeatures").take(3):
 `StringIndexer` encodes a string column of labels to a column of label indices.
 The indices are in `[0, numLabels)`, ordered by label frequencies.
 So the most frequent label gets index `0`.
-If the input column is numeric, we cast it to string and index the string values.
+If the input column is numeric, we cast it to string and index the string 
+values. When following pipeline components such as `Estimator` or 
```

Comment:
LGTM, thanks. Your original text was correct, it's just that somehow it parsed differently at the outset. This looks unambiguous and clear.